templates/cloud-native-kubernetes-usage-report-2021/index.html

```diff
@@ -6993,6 +6993,10 @@ <h2 class="section-heading" id="conclusion">Conclusion</h2>
       <p><a href="https://ubuntu.com/kubernetes/kubernetes-operations-survey" class="p-button--positive">Take the survey</a></p>
 
       <p>This report will be published bi-annually, a few weeks after every KubeCon event. The next survey iteration is planned for the week of KubeCon NA 2021, and the new report will be launched in November 2021.</p>
+
+      <p>Finally, if you liked what you read here, feel free to share the link to this page or to it&rsquo;s PDF version.</p>
```

Comment:
Doesn't need the apostrophe - I'll comment on the doc too
```suggestion
      <p>Finally, if you liked what you read here, feel free to share the link to this page or to its PDF version.</p>
```